lib/ember-app.js

```diff
@@ -153,7 +153,13 @@ function serializeHTML(doc, rootElement) {
 function buildWhitelistedRequire(whitelist, distPath) {
   return function(moduleName) {
     if (whitelist.indexOf(moduleName) > -1) {
-      return require(path.join(distPath, 'node_modules', moduleName));
+      try {
+        // Attempt to see if it's a built-in module
+        return require(moduleName);
+      } catch (e) {
+        // See if it's an npm module, since it wasn't built in
+        return require(path.join(distPath, 'node_modules', moduleName));
```

Comment:
I'd rather resolve from this location instead (via resolve package). Also, is distPath guaranteed to have a leading '.' or '/'? If not then I'm not sure his will work..